### PR TITLE
[zh-ch]add Environment, Universes, and Mirrors #1808

### PR DESCRIPTION
**Pull request by jxnu-liguobin**

### CODE REVIEW

**Comment by eed3si9n** [comment]
@jxnu-liguobin Thanks for the contribution!

I can't review this myself so could you find someone who can? /cc @scala/docs-zh

File: _zh-cn/overviews/reflection/environment-universes-mirrors.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+num: 2
+language: zh-cn
+---
+
+<span class="label important" style="float: right;">EXPERIMENTAL</span>
+
+## Environment
+
+反射环境根据反射工作是在运行时还是在编译时完成而有所不同。在运行时和编译时使用的环境之间的区别被封装在一个所谓的 *原型（universe）* 中。
+反射环境的另一个重要方面是我们可以反射的访问一组实体。这组实体由所谓的 *镜像（mirror）* 决定。
```
Comment (liufengyun) [inline _zh-cn/overviews/reflection/environment-universes-mirrors.md:16]: 这个 `原型（universe）` 的翻译有参考出处么？

universe 字面翻译为 `世界`或`宇宙`，虽然在这里都不是很地道。

/cc: @hepin1989 这个在Scala书中有标准翻译么？

**Review by liufengyun — APPROVED** [review]
Otherwise, LGTM

File: _zh-cn/overviews/reflection/environment-universes-mirrors.md (hunk 1 above)
Comment (eed3si9n) [reply _zh-cn/overviews/reflection/environment-universes-mirrors.md:16]: It would be badass if you translated to 天地 :)

Comment (jxnu-liguobin) [reply _zh-cn/overviews/reflection/environment-universes-mirrors.md:16]: hh，叫宇宙吧，我都忘了为什么写成原型了。。

Comment (liufengyun) [reply _zh-cn/overviews/reflection/environment-universes-mirrors.md:16]: 我感觉翻译为 “世界” 可能更好些，虽然世界更常用于翻译`world`。世界的用法相对不突兀，因为在翻译possible worlds semantics里面也得用"世界"这个词。

Comment (eed3si9n) [reply _zh-cn/overviews/reflection/environment-universes-mirrors.md:16]: Isn't multiverse translated as 多重宇宙? If so one of them would be 宇宙。If either of the options sound "out there", maybe a strategy to take could be add a supplemental word in front of it like _linguistic_ universe, or _symbolic_ world: 语言宇宙?

Comment (liufengyun) [reply _zh-cn/overviews/reflection/environment-universes-mirrors.md:16]: I think adding an adjective is a good strategy. I'm not sure what's the case in Japanese, in Chinese, the world `宇宙` usually means the physical time-space universe, while `世界` could also naturally denote logical spaces in addition to the physical time-space. Therefore, I think `世界` is a little better than `宇宙` in Chinese.

**Pull request merged**